roles/openshift_hosted/templates/registry_config.j2

```diff
@@ -13,6 +13,9 @@ storage:
     accesskey: {{ openshift.hosted.registry.storage.s3.accesskey }}
     secretkey: {{ openshift.hosted.registry.storage.s3.secretkey }}
     region: {{ openshift.hosted.registry.storage.s3.region }}
+{%   if 'regionendpoint' in openshift.hosted.registry.storage.s3 %}
+    regionendpoint: {{ openshift.hosted.registry.storage.s3.regionendpoint }}
+{%   endif -%}
```

Comment:
It appears a line spacing issue occurs with this in place. For example

```
    region: us-east-1
bucket: rcook-pr-test
```

@detiber noted that it was the - on line 18